docs/core/porting/index.md

```diff
@@ -13,35 +13,91 @@ You might have code that currently runs on the .NET Framework that you're intere
+1. Use the [.NET Portability Analyzer](../../standard/analyzers/portability-analyzer.md) to analyze your assemblies and see if they're portable to .NET Core.
 
-3. Install the [.NET API analyzer](../../standard/analyzers/api-analyzer.md) into your projects to identify APIs that throw <xref:System.PlatformNotSupportedException> on some platforms and some other potential compatibility issues.
+   The API Portability Analyzer tool analyzes your compiled assemblies and generates a report. This report shows a high-level portability summary and a breakdown of each API you're using that isn't available on NET Core. While using the tool, only submit the individual project you are converting to focus on the API changes that are potentially needed. Many of the APIs have equivalent availability in .NET Core, which you'll want to switch to.
+
+   There are some cases where APIs are not equivalent and you'll need to do some compiler preprocessor directives (i.e. `#if NET45`) to special case the platforms. At this point, you're project will still be targeting .NET Framework. For each of these targeted cases, it is recommended to use well-known conditionals that can be understood as a scenario.  For example, AppDomain support in .NET Core is limited, but for the scenario of loading and unloading assemblies, there is a new API that is not available in .NET Core. A common way to handle this in code would be something like this:
+
+   ```csharp
+   #if FEATURE_APPDOMAIN_LOADING
+   // Code that uses appdomains
+   #elif FEATURE_ASSEMBLY_LOAD_CONTEXT
+   // Code that uses assembly load context
+   #else
+   #error Unsupported platform
+   #endif
+   ```
+
+1. Install the [.NET API analyzer](../../standard/analyzers/api-analyzer.md) into your projects to identify APIs that throw <xref:System.PlatformNotSupportedException> on some platforms and some other potential compatibility issues.
 
    This tool is similar to the portability analyzer, but instead of analyzing if code can build on .NET Core, it analyzes whether you're using an API in a way that will throw a <xref:System.PlatformNotSupportedException> at run time. Although this isn't common if you're moving from .NET Framework 4.7.2 or higher, it's good to check. For more information about APIs that throw exceptions on .NET Core, see [APIs that always throw exceptions on .NET Core](../compatibility/unsupported-apis.md).
 
-4. Convert all of your `packages.config` dependencies to the [PackageReference](/nuget/consume-packages/package-references-in-project-files) format with the [conversion tool in Visual Studio](/nuget/consume-packages/migrate-packages-config-to-package-reference).
+1. If you are using Windows-specific APIs (such as registry access), you can install the [Windows Compatibility Pack](./windows-compat-pack.md).
+
+1. At this point, you can switch to targeting .NET Core (generally for applications) or .NET Standard (for libraries).
```

Comment:
Should we discuss in any more detail how a user decides whether to target NetCore or NetStandard? It may be out of scope for this doc, but I could also imagine a sentence or two explaining that apps need to use NetCore whereas it's a best practice to target libraries at NetStandard for portability unless there are dependencies (recent ASP.NET Core or EF Core libraries, for example) that require NetCore.

Reply:
I've added some text for that, but couldn't find a link. Do you know of docs that address that choice?

Reply:
Now that you mention it, I don't know of any docs on that topic. May need to just give a quick description here for the moment. In the future, that may be something we want to add somewhere in the .NET docs.

Reply:
I've added some text, and agree we should look into a more detailed page about it